--- orange-0.3/Makefile.am	2003-08-17 15:06:40.000000000 +0300
+++ orange-0.3-pc/Makefile.am	2007-11-12 02:27:18.126416722 +0200
@@ -6,3 +6,5 @@
 
 SUBDIRS = lib src .
 
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = liborange.pc
--- orange-0.3/configure.ac.in	2005-04-09 12:01:40.000000000 +0300
+++ orange-0.3-pc/configure.ac.in	2007-11-12 02:29:20.718314556 +0200
@@ -120,6 +120,7 @@
     ])
 
 AC_OUTPUT([Makefile
+		   liborange.pc
            lib/Makefile
            src/Makefile])
 
--- /dev/null	2006-03-28 14:00:37.000000000 +0300
+++ orange-0.3-pc/liborange.pc	2007-11-12 02:30:47.419656780 +0200
@@ -0,0 +1,11 @@
+prefix=/usr
+exec_prefix=/usr
+libdir=/usr/lib
+includedir=/usr/include
+
+Name: liborange
+Description: Library to extract CAB files from self-extracting installers
+Version: 0.3
+Requires: libsynce libdynamite libunshield
+Libs: -L${libdir} -lorange
+Cflags: -I${includedir}
